Formula/q/qrcp.rb

```diff
@@ -24,7 +24,7 @@ class Qrcp < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", *std_go_args
+    system "go", "build", *std_go_args(ldflags: "-s -w -X main.version=#{version}")
```

Comment:
This is not in `main` package, but a version package:
https://github.com/claudiodangelis/qrcp/blob/0a0bd6079e533af2ac3c414efff00f580509808a/version/version.go#L7

```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w -X github.com/claudiodangelis/qrcp/version.version=#{version}")
```

a version test would be useful here.

Reply:
See also https://github.com/claudiodangelis/qrcp/blob/0a0bd6079e533af2ac3c414efff00f580509808a/.goreleaser.yml#L8

Reply:
make sense 👍